utils/schemed-kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/schemedkafka/schemaregistry/universal/ParsedSchemaSupport.scala

```diff
@@ -96,16 +101,20 @@ object JsonSchemaSupport extends ParsedSchemaSupport[OpenAPIJsonSchema] {
 
   override def typeDefinition(schema: ParsedSchema): TypingResult = schema.cast().returnType
 
-  override def extractSinkValueParameter(schema: ParsedSchema)(implicit nodeId: NodeId): ValidatedNel[ProcessCompilationError, SinkValueParameter] =
-    JsonSinkValueParameter(schema.cast().rawSchema(), defaultParamName = SinkValueParamName)
+  override def extractSinkValueParameter(schema: ParsedSchema, rawMode: Boolean, validationMode: ValidationMode, rawParameter: Parameter)(implicit nodeId: NodeId): ValidatedNel[ProcessCompilationError, SinkValueParameter] = {
+    if (rawMode) {
+      Validated.Valid(
+        SinkSingleValueParameter(rawParameter, new JsonSchemaOutputValidator(validationMode, schema.cast().rawSchema(), schema.cast().rawSchema()))
+      )
+    } else {
+      JsonSinkValueParameter(schema.cast().rawSchema(), defaultParamName = SinkValueParamName, ValidationMode.lax)
```

Comment:
Please add the comment why lax is hardcoded here. Especially that validationMode is available here